.github/workflows/scala.yml

```
@@ -0,0 +1,36 @@
+name: Scala Test
+
+on:
+  push:
+    branches: [ "scala*" ]
+
+jobs:
+  # TEST
+  test-scala-coding:  
```

yamllint

[error] trailing spaces

(trailing-spaces)